### PR TITLE
Breakout widgets into separate modules.

### PR DESCRIPTION
**Pull request by mcdo0486**
Breakout widgets into separate modules. Auto-import current modules at pymeasure/display/widgets/__init__.py for backwards compatibility

### CODE REVIEW

**Comment by msmttchr** [comment]
> Breakout widgets into separate modules. Auto-import current modules at pymeasure/display/widgets/**init**.py for backwards compatibility

Thanks for your proposal. I like the idea. I would suggest to run `flake8` in your local PC before pushing the changes, otherwise you could have a lot of iterations before converging (this happened to me as well).

**Comment by mcdo0486** [comment]
> > Breakout widgets into separate modules. Auto-import current modules at pymeasure/display/widgets/**init**.py for backwards compatibility
> 
> Thanks for your proposal. I like the idea. I would suggest to run `flake8` in your local PC before pushing the changes, otherwise you could have a lot of iterations before converging (this happened to me as well).

Thanks for that tip. I think all linting issues are caught now, I'll definitely use flake8 next time.

**Review by msmttchr — COMMENTED** [review]
@mcdo0486, this looks good to me, even though I am not entirely sure I can understand all the ramifications of the changes.
I wonder, if there could be any issue of backward compatibility. Do you have any comment on this ?
Before merging, I would prefer to have a second opinion from other maintainers.

**Comment by msmttchr** [comment]
@CasperSchippers and @bilderbuchi, at your convenience, could you please have a look at this PR?

**Comment by bilderbuchi** [comment]
Yes I will. It's the "convenience" that I'd hard to come by. 😝

**Comment by CasperSchippers** [comment]
I like the idea of splitting the file into separate modules.

I'll try having a look tomorrow evening (sorry for the wait, as mentioned already, time is hard to come by).

One concern I can already raise from a git point of view: splitting a file causes us to lose the history/blame of the code, right? I think there are ways to ensure we keep the history, though from top of my head I don't know how easy/difficult that is and if it's worth the effort (also a question to @bilderbuchi and @msmttchr); I myself do sometimes use the history, but not very often.

**Comment by bilderbuchi** [comment]
Git blame tracks moved/renamed files. There is a [flag](https://stackoverflow.com/a/10664943/599884) to track moved code portions across files, as well:  [`-C`](https://git-scm.com/docs/git-blame#Documentation/git-blame.txt--Cltnumgt).
In any case, if I had to choose, I'd rather argue for better-organised code even at the cost of some history features.

File: pymeasure/display/widgets/__init__.py
Comment (CasperSchippers) [inline pymeasure/display/widgets/__init__.py:1]: Could you also add the copyright text over here

**Review by CasperSchippers — APPROVED** [review]
I've had a thorough look and looks good to me; no issues that I could find. Only question is to add the copyright text to the init file

**Review by msmttchr — APPROVED** [review]

File: pymeasure/display/widgets/__init__.py
Comment (mcdo0486) [reply pymeasure/display/widgets/__init__.py:25]: Yup pushed that up

**Pull request merged**
